models/custom-application/src/custom-application-installation/custom-application-installation/generator.ts

```diff
@@ -0,0 +1,24 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import * as CustomApplicationInstallationPermission from '../../custom-application-installation-permission';
+import type { TCustomApplicationInstallation } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TCustomApplicationInstallation>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    createdAt: fake(getOlderDate),
+    updatedAt: fake(getNewerDate),
+    acceptedPermissions: fake(() => [
+      CustomApplicationInstallationPermission.presets.ViewOnlyPermissions(),
+      CustomApplicationInstallationPermission.presets.ManageOnlyPermissions(),
+    ]),
+    projects: fake(() => []), //TODO: add ProjectExtension
```

Comment:
This seems to be missing. You also need to update the transformer

Reply:
I think the `ProjectExtension` is implemented here: https://github.com/commercetools/test-data/tree/main/models/project/src/mc-project

Both `OrganizationExtension` and `ProjectExtension` entities have been named with the `MC` prefix.

Reply:
> I think the ProjectExtension is implemented here
> Both `OrganizationExtension` and `ProjectExtension` entities have been named with the `MC` prefix.

No these refer to the MC GraphQL API (gateway), for example [Project](https://github.com/commercetools/merchant-center-services/blob/fda3f3bdd6f5bdb6df219778e4aac24952660cd7/packages/gateway/lib/graphql/user/user.schema.graphql#L93-L121).

> We don't have `ProjectExtension` Model yet that's why. We can update it when implemented.

You're right, I thought we already had those migrated. Thanks for checking

Reply:
💯